debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/SinkConnectorIT.java

```diff
@@ -0,0 +1,122 @@
+        try {
+            TestInfrastructureHelper.getDebeziumContainer().waitForStreamingRunning("mysql", config.getString(CommonConnectorConfig.TOPIC_PREFIX));
+        }
+        catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
+        TestInfrastructureHelper.getDebeziumContainer().deleteConnector(SOURCE_CONNECTOR_NAME);
+    }
+
+    private Configuration.Builder mongodbSinkConfig() {
+        return Configuration.create()
+                .with("connector.class", MongoDbSinkConnector.class)
+                .with(MongoDbSinkConnectorConfig.CONNECTION_STRING, getMongoDbDeployment().getConnectionString())
+                .with(SinkConnectorConfig.TOPICS_REGEX_CONFIG, "dbserver1\\.inventory\\..*")
+                .with(MongoDbSinkConnectorConfig.SINK_DATABASE, DATABASE_NAME);
+    }
+
+    default void startSink(Function<Configuration.Builder, Configuration.Builder> custConfig) {
+        TestInfrastructureHelper.setupDebeziumContainer(Module.version(), null);
+        TestInfrastructureHelper.startContainers(DATABASE.DEBEZIUM_ONLY);
+        final Configuration config = custConfig.apply(mongodbSinkConfig()).build();
+        TestInfrastructureHelper.getDebeziumContainer().registerConnector(SINK_CONNECTOR_NAME, ConnectorConfiguration.from(config.asMap()));
+        TestInfrastructureHelper.getDebeziumContainer().ensureConnectorTaskState(SINK_CONNECTOR_NAME, 0, Connector.State.RUNNING);
+    }
+
+    static void stopContainers(MongoDbDeployment mongo) {
+        TestInfrastructureHelper.stopContainers();
+        if (null != mongo) {
+            mongo.stop();
+        }
+    }
+
+    default List<String> listCollections(String dbName) {
+        try (var client = TestHelper.connect(getMongoDbDeployment())) {
+            MongoDatabase db1 = client.getDatabase(dbName);
+            var collections = db1.listCollectionNames();
+            collections.forEach((var collectionName) -> {
+                LOGGER.info("Found collection '{}' from database '{}'", collectionName, dbName);
+                if (LOGGER.isDebugEnabled()) {
+                    LOGGER.debug("with documents:");
+                    db1.getCollection(collectionName).find().forEach((var doc) -> LOGGER.debug("Found document: {}", doc));
+                }
+            });
+            return collections.into(new ArrayList<>());
+        }
+    }
+
+    default void checkSinkConnectorWritesRecords() {
+        startSink(Function.identity());
+        Awaitility.await()
+                .atMost(TestHelper.waitTimeForRecords() * 5L, TimeUnit.SECONDS)
+                .until(() -> listCollections(DATABASE_NAME).size() >= 6);
+        List<String> collections = listCollections(DATABASE_NAME);
+        LOGGER.debug("List collections: {}", Arrays.toString(collections.toArray()));
+        Assertions.assertThat(listCollections(DATABASE_NAME).size()).isEqualTo(6);
```

Comment:
@rk3rn3r I think we can use `assertj` only. There can be for example
```
Assertions.assertThat(listCollections(DATABASE_NAME)).hasSize(6);
        Assertions.assertThat(collections).containsExactlyInAnyOrder("dbserver1_inventory_addresses", "dbserver1_inventory_orders", "dbserver1_inventory_customers",
                "dbserver1_inventory_products_on_hand", "dbserver1_inventory_geom", "dbserver1_inventory_products");
```

Reply:
Why is that

> we can use `assertj` only

?

Reply:
Wy should we use both hamcrest and assertj in a single class if they offer the same functionality?

Reply:
done